### DISCUSSION TITLE
Mac battery optimisation + Al Dente (Free)

### DISCUSSION THREAD

**Discussion by gsohoni** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1122] [Q&A] [answered]
Hi

Apologies if this has been clarified earlier.

On Sonoma, MBP, could one keep the Mac's battery optimisation on along with AlDente? The most recent free version doesn't advice you to turn off Mac's default battery optimisation upon install.

The default Mac battery optimisation takes a few days or more to kick in and stop the battery charging at around 80% and if you deviate from the usual pattern, this could again take a few days to return to charging only till 80%.

What I would like to understand is if I keep AlDente enabled and keep the battery charged at 80% (from the moment AlDente is enabled), would the mac optimisation still continue to learn my usage pattern? And could I switch to just the Mac battery optimisation if AlDente doesn't seem to make any noticeable impact on battery cycles/health?

Thanks for providing these details.

Keep up the good work.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @gsohoni,

> On Sonoma, MBP, could one keep the Mac's battery optimisation on along with AlDente?

No, this is not recommended. If you have two different programs using the same technical pathways and doing similar things, they will start interfering with each other and none will work correctly. However, the behavior tends to be rather inconsistent when both are activated.

> The most recent free version doesn't advice you to turn off Mac's default battery optimization upon install.

This is not correct. One of the first pages during the installation process indicates to disable Apple's Optimized Battery Charging. However, if you have parts of AlDente already installed, these steps during the installation process might be skipped.

In general, we are not recommending using AlDente and Apple's Optimized Battery Charging at the same time due to inconsistent behavior. While AlDente is used and Optimized Battery Charging is disabled, macOS will not learn the user's charging behavior. When AlDente is removed and Optimized Battery Charging is enabled again, macOS will learn the user's behavior again from the beginning.

Best,
Matthias

**Reply by gsohoni** [reply]
Thanks for providing clarity on this @MatthiasKerbl. Appreciate you coming back on this.
